#Point1
float32 x1
float32 y1
#int64 z1

#Point2
float32 x2
float32 y2
#int64 z2
